### ISSUE TITLE
🚑 Trajectory data table broken

### ISSUE THREAD

**Issue by shankari** [e-mission/op-admin-dashboard#109]
Error logs:

```
ERROR:app_sidebar_collapsible:Exception on /_dash-update-component [POST]
Traceback (most recent call last):
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/flask/app.py", line 2529, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/dash/dash.py", line 1310, in dispatch
    ctx.run(
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/dash/_callback.py", line 442, in add_context
    output_value = func(*func_args, **func_kwargs)  # %% callback invoked %%
  File "/usr/src/app/pages/data.py", line 123, in render_content
    store_trajectories = update_store_trajectories(start_date, end_date, timezone, store_excluded_uuids)
  File "/usr/src/app/pages/data.py", line 44, in update_store_trajectories
    df = query_trajectories(start_date, end_date, tz)
  File "/usr/src/app/utils/db_utils.py", line 170, in query_trajectories
    df['data.mode_str'] = df['data.mode'].apply(lambda x: ecwm.MotionTypes(x).name if x in set(enum.value for enum in ecwm.MotionTypes) else 'UNKNOWN')
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/pandas/core/series.py", line 4771, in apply
    return SeriesApply(self, func, convert_dtype, args, kwargs).apply()
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/pandas/core/apply.py", line 1123, in apply
    return self.apply_standard()
  File "/root/miniconda-23.5.2/envs/emission/lib/python3.9/site-packages/pandas/core/apply.py", line 1174, in apply_standard
    mapped = lib.map_infer(
  File "pandas/_libs/lib.pyx", line 2924, in pandas._libs.lib.map_infer
  File "/usr/src/app/utils/db_utils.py", line 170, in <lambda>
    df['data.mode_str'] = df['data.mode'].apply(lambda x: ecwm.MotionTypes(x).name if x in set(enum.value for enum in ecwm.MotionTypes) else 'UNKNOWN')
AttributeError: module 'emission.core.wrapper.modeprediction' has no attribute 'MotionTypes'
```

**Comment by shankari** [comment]
https://github.com/e-mission/op-admin-dashboard/assets/2423263/7ce2c35e-f34e-4480-8502-871f002d1fb0

**Comment by shankari** [comment]
@jiji14 @achasmita for visibility

**Comment by achasmita** [comment]
I got the same error running locally ```AttributeError: module 'emission.core.wrapper.modeprediction' has no attribute 'MotionTypes'```.  Removing ```import emission.core.wrapper.modeprediction as ecwm```  (as we are not using it any longer) solved the issue.

**Comment by shankari** [comment]
@achasmita great@ Can you submit a PR for this fix? Did you also look into #110?

**Comment by shankari** [comment]
Fixed in #111
